opentelemetry/chart/values.yaml

```diff
@@ -17,14 +17,15 @@ opentelemetry-operator:
     failurePolicy: 'Ignore'
     create: true
     certManager:
-      enabled: false
-    autoGenerateCert:
       enabled: true
+      duration: "17280h"
+    autoGenerateCert:
+      enabled: false
       recreate: false
```

Comment:
Same as for the comment below. These values are part of the PluginDefinition's `options` and should be configured directly on the Plugin.

Reply:
I have removed these defaults.